portal-user-datastore-common-test/src/main/java/com/telecominfraproject/wlan/portaluser/datastore/BasePortalUserDatastoreTest.java

```diff
@@ -175,6 +175,39 @@ public void testGetAllInSet() {
 
     }
     
+    @Test
+    public void testGetCustomerIdsForUsername() {
+        Set<PortalUser> createdSet = new HashSet<>();
+        List<Integer> checkCustomerIdsList = new ArrayList<>();
+        String testUsername = "testSameUsername";
+
+        //Create test PortalUsers
+        PortalUser portalUser = new PortalUser();
+        portalUser.setUsername(testUsername);
+
+        for (int i = 0; i < 10; i++) {
+            portalUser.setCustomerId(i);
```

Comment:
It's a tricky thing - the unit tests from the same test suite class can be executed in parallel. 
In that case they may affect each other - object created in the datastores by one test may be picked up by other tests, making them fail because object counts are different, missing objects re-appear, etc.

So it is better for each test to use its own unique ids for customer/equipment/location/etc.
In this particular case:
       portalUser.setCustomerId((int) testSequence.incrementAndGet());